arangod/Cluster/ClusterInfo.cpp

```diff
@@ -617,30 +611,23 @@ void ClusterInfo::loadPlan() {
   // we will set in the end
 
   std::shared_ptr<arangodb::velocypack::Builder> planBuilder;
-  std::tie(planBuilder, idx) = agencyCache.get(prefixPlan);
-  auto planSlice = planBuilder->slice();
+  std::tie(planBuilder, idx) = agencyCache.get("Plan/Version");
+  auto planVersionSlice = planBuilder->slice();
 
-  if (!planSlice.isObject()) {
-    LOG_TOPIC("bc8e1", ERR, Logger::CLUSTER)
-        << "\"Plan\" is not an object in agency";
-    return;
-  }
-
-  auto planVersionSlice = planSlice.get("Version");
-
-  if (planVersionSlice.isNumber()) {
+  if (idx > 0 && planVersionSlice.isNumber()) {
     try {
       newPlanVersion = planVersionSlice.getNumber<uint64_t>();
-    } catch (...) {
-    }
+    } catch (...) { }
   }
+  planBuilder->clear();
 
   LOG_TOPIC("c6303", TRACE, Logger::CLUSTER) << "loadPlan: newPlanVersion=" << newPlanVersion;
 
   if (newPlanVersion == 0) {
-    LOG_TOPIC("d68ae", WARN, Logger::CLUSTER)
+    LOG_TOPIC("d68ae", DEBUG, Logger::CLUSTER)
```

Comment:
If we see this warning at startup it is a sign that we have to `waitForPlan(1)` somewhere. In other cases, I think we want to retain it as `WARN`.